apps/storefront/src/pages/Registered/RegisteredAccount.tsx

```diff
@@ -116,12 +110,17 @@ export default function RegisteredAccount(props: RegisteredAccountProps) {
     'email';
 
   const validateEmailValue = async (email: string) => {
+    const isRegisterAsB2BUser = accountType === '1';
     try {
       showLoading(true);
-      const { userType, userInfo: { companyName = '' } = {} } = isB2BUser
+      const { userType, userInfo: { companyName = '' } = {} } = isRegisterAsB2BUser
         ? await checkUserEmail({ email, channelId })
         : await checkUserBCEmail({ email, channelId });
 
+      const isValidUserType = isRegisterAsB2BUser
+        ? [UserTypes.DOESNT_EXIST].includes(userType)
+        : ![UserTypes.B2C].includes(userType);
```

Comment:
Pretty sure these two are the same. Might be worth rebasing based on the other PR. Or waiting for it to be merged 🤔 


https://github.com/bigcommerce/b2b-buyer-portal/pull/38/files/66d1b3562311f09d3ddbc5e90ef49dba4889ff47#diff-5b44a3f9409bc0ff9b10220843f338b25edd67c2faa19d08d88e8d543cd718d2